addons/web/static/src/core/select_menu/select_menu.js

```diff
@@ -64,6 +66,7 @@ export class SelectMenu extends Component {
         searchable: { type: Boolean, optional: true },
         searchPlaceholder: { type: String, optional: true },
         value: { optional: true },
+        multiSelect: { type: Boolean, optional: true },
```

Comment:
this props must be tested

Reply:
and documented

Reply:
I'll write a PR of documentation to update SelectMenu and Notebook with the newest props available